src/js/script.js

```diff
@@ -0,0 +1,211 @@
+const { remote } = require("electron");
+const { Notification }=remote;
+
+//*** utility functions ***
+
+function padzero(num) {
+  var s = "00" + num;
+  return s.substr(s.length - 2);
+}
+function showstatus(session_status,perio_num) {
+  if (session_status == 0) {
+    document.getElementById("status").innerText = "Pomodoro"+perio_num;
+  }
+  if (session_status == 1 || session_status == 4) {
+    document.getElementById("status").innerText = "working..."+perio_num;
+  }
+  if (session_status == 2 || session_status == 5) {
+    document.getElementById("status").innerText = "chilling..."+perio_num;
+  }
+}
+function miniclock(secs) {
+  var min = Math.floor(secs / 60);
+  var sec = Math.floor(secs % 60);
+  document.getElementById("count_down").innerHTML =
+    padzero(min) + ":" + padzero(sec);
+}
+
+///***************************
+
+function getsessioninfo(){
+  //get and display latest setting on load 
+  var work_insec = localStorage.getItem('work_ls') || 25*60;
+  var sbreak_insec = localStorage.getItem('sbreak_ls') || 5*60;
+  var lbreak_insec = localStorage.getItem('lbreak_ls') || 15*60;
+  var work_periods = localStorage.getItem('periods_ls') || 4;
+  document.getElementById("worktime").innerText = "Work Session: "+ work_insec/60 + "mins";
+  document.getElementById("shortbreak").innerText = "Short Break: "+ sbreak_insec/60 + "mins";
+  document.getElementById("longbreak").innerText = "Long Break: "+ lbreak_insec/60 + "mins";
```

Comment:
```suggestion
  document.getElementById("worktime").innerText = "Work Session: "+ work_insec/60 + " mins";
  document.getElementById("shortbreak").innerText = "Short Break: "+ sbreak_insec/60 + " mins";
  document.getElementById("longbreak").innerText = "Long Break: "+ lbreak_insec/60 + " mins";
```

Reply:
In the future we should split this up so that we have a special function that takes a time in seconds and formats it in minutes, rather than repeating this. We can take care of that later, though.